### ISSUE TITLE
Add exercises for STAC

### ISSUE THREAD

**Issue by rogerkuou** [esciencecenter-digital-skills/Lesson-FAIR-Data-Climate#46]
As mentioned in #42, we can add an exercise of "downloading data from a STAC catalogue using the intake library". This can be a dedicated episode after the FAIR data introduction. It can be used as an practical example of FAIR principles.

In this exercise we can cover:

- Retrieve a `Image Collection` via STAC, I am thinking of using [Sentinel-2](https://stacindex.org/catalogs/sentinel-2-and-landsat-8-catalog#/dZEdtMUofyH2kUoQwJh?t=bands) as an example.
- Evaluate this procedure w.r.t. the FAIR principles. 
- Create a data product  (e.g. an NDVI map) using this image collection, and create STAC-complaint metadata for this data set.

**Comment by rogerkuou** [comment]
@Peter9192 @fnattino I added some outlines for the STAC exercise task. Would you like to give some opinions?

**Comment by Peter9192** [comment]
Yes, I like the idea very much. Maybe a good start would be to write some (draft) key points/questions/objectives for this episode, e.g. "How does STAC contribute to a FAIR workflow?" as a question or , "Evaluate the FAIRness of (Sentinal 2) data" as an objective,  etc.

We should make sure to keep track of the packages needed and other prerequisites. I guess I'll have more suggestions once you have a draft PR.

**Comment by fnattino** [comment]
Hi @rogerkuou, sounds great! 

I like the idea of using Sentinel-2 data. I am not sure about the data availability for the catalog that you have linked, in case data would be accessible only as "requester pays", you can always use the [L2A-COGs catalog](https://stacindex.org/catalogs/earth-search#/QgYM1zmboMQu9vwy9DVPQYy3u1cgcvxMGcNBptksS?t=items) that is open.

Also creating the data product seems a very nice idea! The NDVI requires the high-resolution bands - right? Maybe let's just keep in mind that these files might be quite big for the actual computation.

**Comment by rogerkuou** [comment]
> Hi @rogerkuou, sounds great!
> 
> I like the idea of using Sentinel-2 data. I am not sure about the data availability for the catalog that you have linked, in case data would be accessible only as "requester pays", you can always use the [L2A-COGs catalog](https://stacindex.org/catalogs/earth-search#/QgYM1zmboMQu9vwy9DVPQYy3u1cgcvxMGcNBptksS?t=items) that is open.
> 
> Also creating the data product seems a very nice idea! The NDVI requires the high-resolution bands - right? Maybe let's just keep in mind that these files might be quite big for the actual computation.

Thx for the suggestion, you are totally right about the availability. I will use the catalog you suggested. 
For NDVI, I will see how it goes. In principle it's just computation using multi bands, and actually it can be done with a very small crop.
